<?xml version="1.0"?>

<launch>
  <!-- rusrun l298n l298n_node -->
  <node pkg="l298n" type="l298n_node" name="motor_driver_node" respawn="true" output="log">
    <param name="IN1" type="int" value="21" />
    <param name="IN2" type="int" value="20" />
    <param name="IN3" type="int" value="23" />
    <param name="IN4" type="int" value="24" />
    <param name="ENA" type="int" value="12" />
    <param name="ENB" type="int" value="13" />
    <param name="minimum_motor_command" type="int" value="20" />
    <param name="maximum_motor_command" type="int" value="300" />
  </node>

</launch>

